docs/src/RadioButton.doc.js

```diff
@@ -50,32 +50,148 @@ function RadioButtonExample() {
     >
       <Box paddingY={1}>
         <RadioButton
-          checked={gender === 'male'}
-          id="genderMale"
-          label="Male"
-          name="gender"
-          onChange={() => setGender( 'male' )}
-          value="male"
+          checked={favorite === 'dogs'}
+          id="favoriteDog"
+          label="Dogs"
+          name="favorite"
+          onChange={() => setFavorite( 'dogs' )}
+          value="dogs"
         />
       </Box>
       <Box paddingY={1}>
         <RadioButton
-          checked={gender === 'female'}
-          id="genderFemale"
-          label="Female"
-          name="gender"
-          onChange={() => setGender( 'female' )}
-          value="female"
+          checked={favorite === 'cats'}
+          id="favoriteCat"
+          label="Cats"
+          name="favorite"
+          onChange={() => setFavorite( 'cats' )}
+          value="cats"
         />
       </Box>
       <Box paddingY={1}>
         <RadioButton
-          checked={gender === 'other'}
-          id="genderOther"
-          label="Other"
-          name="gender"
-          onChange={() => setGender( 'other' )}
-          value="other"
+          checked={favorite === 'plants'}
+          id="favoritePlants"
+          label="Plants"
+          name="favorite"
+          onChange={() => setFavorite( 'plants' )}
+          value="plants"
+        />
+      </Box>
+    </Box>
+  );
+}
+`}
+  />
+);
+
+card(
+  <Example
+    description="
+    Here is an example of a group of radio buttons with subtext.
```

Comment:
NIT: No need to say `Here is an example of a ...` - instead we could just name it `A group of radio buttons with subtext`